examples/haas.cfg

```diff
@@ -62,10 +62,20 @@ endpoint = http://127.0.0.1:5000
 
 [database]
 # A SQLAlchemy database URI, specifying the database for the server to
-# connect to. At present, only SQLite and PostgreSQL are supported.
+# connect to. At present, only SQLite and PostgreSQL are supported, and only
+# PostgreSQL is supported by our migration scripts.
+#
+# If a relative path is used with SQLite, it will be relative to the HaaS source
+# tree. Unless you're doing development, this is almost certainly not what you
+# want, so use an absolute path.
+#
+# Use the sqite database file ``/absolute/path/to/haas.db``:
+uri = sqlite:////absolute/path/to/haas.db
```

Comment:
So I don't feel super strongly about that specific option, but some points:

1. That file's purpose is to serve as a thorough reference; we have another file in that directory that works as an out-of-the-box config for developers (mentioned at the top of this file), and in production there's really no way to provide something like that (besides, we encourage using postgres anyway).
2. My reasoning was that per the comments above, storing the database in the source tree isn't something you want unless you're developing, and most of the options in this file are more production-oriented. Besides, as I mentioned, we already have an off-the-shelf config.

If we are going to change it I'd recommend going for the postgres as the one that's exposed. That doesn't actually address the reason you're asking for it, but that's not really achieveable in a production setting anyway.

I don't feel super strongly; read over the above points and let me know what you think.